dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/program/notification/ProgramNotificationTemplateServiceTest.java

```diff
@@ -108,35 +107,43 @@ void setUp() {
   }
 
   @Test
-  void testGetProgramNotificationTemplates() {
+  void shouldGetProgramNotificationTemplates() {
     ProgramNotificationTemplateOperationParams param =
         ProgramNotificationTemplateOperationParams.builder()
             .program(UID.of(program.getUid()))
             .build();
+
     List<ProgramNotificationTemplate> templates =
         programNotificationTemplateService.getProgramNotificationTemplates(param);
-    assertFalse(templates.isEmpty());
-    assertEquals(2, templates.size());
-    assertTrue(templates.contains(pnt1));
-    assertTrue(templates.contains(pnt2));
-    assertFalse(templates.contains(pnt3));
+
+    assertContainsOnly(List.of(pnt1, pnt2), templates);
```

Comment:
If order matters use `assertEquals(List.of(pnt1, pnt2), templates)` as `assertContainsOnly` does not assert on order. Please check all tests.

Reply:
done